src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -320,9 +327,19 @@ export function CollapsibleNavGroupEnabled({
             hasShadow={false}
             className="eui-yScroll flex-1-container"
           >
-            {shouldShowCollapsedNavHeaderContent && collapsibleNavHeaderRender ? (
+            {shouldShowCollapsedNavHeaderContent &&
+            collapsibleNavHeaderRender &&
+            !currentWorkspace ? (
               <>
-                {collapsibleNavHeaderRender()}
+                <EuiPanel
+                  paddingSize="none"
+                  color="transparent"
+                  hasBorder={false}
+                  hasShadow={false}
+                  style={{ height: '40vh' }}
```

Comment:
```diff
        {!isNavOpen ? null : (
          <EuiPanel
            hasBorder={false}
            borderRadius="none"
            paddingSize={!isNavOpen ? 's' : 'l'}
            hasShadow={false}
            className="eui-yScroll flex-1-container"
+            style={{ display: 'flex', flexDirection: 'column' }}
          >
            {shouldShowCollapsedNavHeaderContent &&
            collapsibleNavHeaderRender &&
            !currentWorkspace ? (
              <>
                <EuiPanel
                  paddingSize="none"
                  color="transparent"
                  hasBorder={false}
                  hasShadow={false}
+                  style={{ overflow: 'hidden' }}
                >
                  {collapsibleNavHeaderRender()}
                </EuiPanel>
+                <EuiSpacer style={{ marginTop: 'auto' }} />
              </>
            ) : null}
```

Reply:
<img width="274" alt="image" src="https://github.com/user-attachments/assets/3f596ea6-bf24-4cca-8b28-2c8a0ffd56b5">

Since you give it a fixed `40vh` height, the left nav still look weird, see screenshot. Please check above code that shared, which make sure the menu at the bottom.

Reply:
<img width="275" alt="image" src="https://github.com/user-attachments/assets/0a555caf-3cda-4ab6-a660-860dcc5ca556">

It should be look like this